src/vcu/src/can_handles.c

```diff
@@ -293,6 +296,37 @@ void handleDashRequest(Frame *msg) {
   control_settings.active_aero_enabled = unpacked_msg.active_aero_enabled;
 }
 
+void handleDashRequestLC(Frame *msg) {
+  can0_DashRequestLC_T unpacked_msg;
+
+  unpack_can0_DashRequestLC(msg, &unpacked_msg);
+
+  // printf("speeding_up_torque: %d\r\n", unpacked_msg.speeding_up_torque);
```

Comment:
Just leaving a comment here to be able to block the PR. We should take a second look at the Dash changes before merging, but for now we roll with this for testing trip.